### DISCUSSION TITLE
166

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#117] [General]
# 潮流周刊第166期 - 猫晒太阳

封面图来源于周末汤圆在阳台晒太阳的照片，天气很不错，猫这种生物很会享受，哪儿舒服呆哪儿。

https://weekly.tw93.fun/posts/166-%E7%8C%AB%E6%99%92%E5%A4%AA%E9%98%B3

**Comment by CY-Christin** [comment]
记账软件我也推荐一个正在用的 [App Store 上的“貔貅记账 - Pixiu”](https://apps.apple.com/cn/app/%E8%B2%94%E8%B2%85%E8%AE%B0%E8%B4%A6-pixiu/id1461452315)，我用下来感觉还不错，也是偏资产管理的。

**Reply by tw93** [reply]
谢谢推荐

**Reply by eric2go** [reply]
同使用pixiu4年多了，价格良心，功能强大但是不繁琐，值得一试

**Comment by yslemmo** [comment]
用一点多的iCost，界面美观简约，也提供买断版本。更新比较勤，另外app智能记账的体验也不错（在付款界面双击后盖，自动截屏记账）

**Reply by tw93** [reply]
谢谢推荐 我有空去试试
